test/ranges/subset.js

```diff
@@ -15,6 +15,12 @@ const cases = [
   ['>2 <1', '3', true],
   ['1 || 2 || 3', '>=1.0.0', true],
 
+  // everything is a subset of *
+  ['1.2.3', '*', true],
+  ['^1.2.3', '*', true],
+  ['^1.2.3-pre.0', '*', true],
```

Comment:
This is incorrect, though.  `^1.2.3-pre.0` is not a subset of `*` unless `includePrerelease` is set, because there are versions that satisfy `^1.2.3-pre.0` and do not satisfy `*`.

```
> semver.satisfies('1.2.3-pre.0', '^1.2.3-pre.0')
true
> semver.satisfies('1.2.3-pre.0', '*')
false
```